frontend/integration-tests/protractor.conf.ts

```diff
@@ -90,6 +90,7 @@ export const config: Config = {
     environment: ['tests/base.scenario.ts', 'tests/environment.scenario.ts'],
     secrets: ['tests/base.scenario.ts', 'tests/secrets.scenario.ts'],
     crud: ['tests/base.scenario.ts', 'tests/crud.scenario.ts', 'tests/secrets.scenario.ts', 'tests/filter.scenario.ts', 'tests/modal-annotations.scenario.ts', 'tests/environment.scenario.ts'],
+    monitoring: ['tests/base.scenario.ts', 'tests/monitoring.scenario.ts'],
```

Comment:
How are these expected to be ran as part of console CI? We should look at deploying `master` of the monitoring stack (Operator?) similar to what we do with OLM.